src/constants.py

```diff
@@ -0,0 +1,31 @@
+# pylint: disable=too-few-public-methods
+
+
+class LabelName:
```

Comment:
The enums here prevent using plain strings throughout the project. If we ever need to change any of them - they are defined in one place.